x-pack/test/reporting_api_integration/reporting_and_security/usage.ts

```diff
@@ -21,7 +21,7 @@ export default function ({ getService }: FtrProviderContext) {
   const reportingAPI = getService('reportingAPI');
   const usageAPI = getService('usageAPI');
 
-  describe('reporting usage', () => {
+  describe('Usage', () => {
```

Comment:
The full name of the test was `Reporting reporting Usage` and this changes it to `Reporting Usage`